apps/etterlatte-utbetaling/src/main/kotlin/utbetaling/avstemming/KonsistensavstemmingJob.kt

```diff
@@ -76,9 +79,19 @@ class KonsistensavstemmingJob(
                                         log.info("Konsistensavstemming er allerede kjoert i dag ($idag)")
                                     }
                                 }
-                                Saktype.OMSTILLINGSSTOENAD -> {
-                                    log.info("Konsistensavstemming for omstillingsstoenad ennaa ikke implementert")
-                                    /* TODO: Blir haandtert i EY-1274 */
+                                Saktype.OMSTILLINGSSTOENAD  -> {
```

Comment:
Samme som grensesnittavstemming, kanskje lettere om vi tar SakType som input her så slipper vi å duplisere logikk - dette er jo egentlig bare en wrapper for å sparke i gang jobben.